synapse/models/economic.py

```diff
@@ -115,12 +115,18 @@ def getModelDefs(self):
                     ('from:pay:card', ('econ:pay:card', {}), {
                         'doc': 'The payment card making the payment.'}),
 
-                    ('to:contact', ('ps:contact', {}), {
-                        'doc': 'Contact information for the person/org being paid.'}),
+                    ('from:coinaddr', ('crypto:currency:address', {}), {
```

Comment:
unclear as to the difference between a payment and a transaction. There might be multiple transactions or multiple payments related to the same exchange of goods.

Reply:
The one part I'd want for bitcoin is the transaction identifier (TXID). That would allow you to tie the payment to the blockchain.

Reply:
By "transaction" i'm going to assume you mean `econ:purchase` ?  The econ model already differentiates between purchase and payment exactly for this reason. :)

Reply:
Agreed that a :btc:txid is probably worth while even in the first pass.  Will add.